mlflow/pyfunc/scoring_server/__init__.py

```diff
@@ -362,6 +354,8 @@ def invocations(data, content_type, model, input_schema):
         )
 
     # Do the prediction
+    # NB: utils._validate_serving_input mimic the scoring process here to validate input_example
```

Comment:
Let's file a follow-up ticket to think about redesigning this conversion processing stage such that model serving transformation logic doesn't live entirely within serving. It would be ideal to have pyfunc utility functions that live within MLflow such that both the serving-compatible example generation and the inference processing layer within the inference server are using the exact same functions.

Reply:
Exactly! I'll file a JIRA ticket :D